tests/validators/base.test.ts

```diff
@@ -0,0 +1,43 @@
+import { s } from '../../src';
+
+describe('BaseValidator', () => {
+	describe('Nullable', () => {
```

Comment:
If a describe references a method name, it should be named the same as the method, including casing.